packages/ag-charts-types/src/chart/chartBuilderOptions.ts

```diff
@@ -137,9 +137,15 @@ export interface ImageDataUrlOptions {
     fileFormat?: string;
 }
 
+// TODO: this is a breaking change as previously it was permissive of any type for each key
+// export interface AgChartState extends AgInitialStateOptions {
+//     version: string;
+// }
+
```

Comment:
We should have been doing this previously, but can't introduce it just yet as it is a breaking change.

Reply:
Add deprecation notice